Makefile.common

```diff
@@ -203,9 +203,17 @@ endif
 	$(QUIET_AS)$(AS) $(ASFLAGS) $(<) -o $@
 
 # Build rule for Windows resource files
-# TODO: Support dependency tracking
+# The regular expression is a portable form of ^(.*\s(FILE|ICON)|#include)\s+"([^"]*)".*$
+# This finds lines in the form '... FILE "path/file.ext" ...' (and also ICON and #include)
+# and replaces them with the $(srcdir)/path/file.ext \
+# The second sed removes winresrc.h (system include) and config.h (not in srcdir, printed
+# by the echo before sed), and strips the trailing backslash on the last line.
 %.o: %.rc
 	$(QUIET)$(MKDIR) $(*D)
+	$(QUIET)$(MKDIR) $(*D)/$(DEPDIR)
+	$(QUIET)echo "$@: $< config.h config.mk \\" > $(*D)/$(DEPDIR)/scummvm.d
+	$(QUIET)sed -n 's:^\(.*[[:space:]]\(FILE\|ICON\)\|#include\)[[:space:]][[:space:]]*"\([^"]*\)".*$$:    $(srcdir)/\3 \\:p; ' $(<) | \
```

Comment:
It would be useful to detect lines containing ``RT_MANIFEST`` and ``DATA`` as well.

Reply:
Done